tests/test_openapi.py

```diff
@@ -286,13 +301,11 @@ def probe(
                 '_Params_probe_': {
                     'properties': {
                         'amount': {
-                            'example': 10,
                             'exclusiveMinimum': 5.0,
                             'title': 'Amount',
                             'type': 'integer',
                         },
                         'data': {
-                            'example': ['111', '222'],
```

Comment:
We miss example, i think it's not a fix actually

Reply:
agree